### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by a1346054**

### CODE REVIEW

**Review by ZoltanFridrich — DISMISSED** [review]
Thank you for this PR.
Everything looks good to me.

**Comment by ZoltanFridrich** [comment]
maybe you will need to squash the commits

**Comment by a1346054** [comment]
Sure, I can squash the commits, isn't that something that's generally frowned upon though? Or was there some sort of test failing in between?

**Comment by a1346054** [comment]
Rebased on latest master.

File: LICENSE [hunk 1]
```diff
@@ -290,8 +290,8 @@ to attach them to the start of each source file to most effectively
 convey the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    {description}
-    Copyright (C) {year}  {fullname}
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (wcampbell0x2a) [inline LICENSE:293]: You should probably fill this in.

Comment (a1346054) [reply LICENSE:293]: It is not meant to be filled in, it's simply a part of the license that informs distributors of changed code how they may wish to describe the changed program and add a copyright statement, as described in https://www.gnu.org/licenses/old-licenses/gpl-2.0-faq.html#GPLOmitPreamble

Comment (wcampbell0x2a) [reply LICENSE:293]: Oh cool, TIL.

Comment (a1346054) [reply LICENSE:293]: That is why, perhaps somewhat amusingly, every GPL2.0 license mentions "Ty Coon, President of Vice"

**Review by Cropi — APPROVED** [review]

**Pull request merged**